rt/transports/http/src/main/java/org/apache/cxf/transport/http/Headers.java

```diff
@@ -580,4 +580,8 @@ public static String toHttpDate(Date date) {
     public static String toHttpLanguage(Locale locale) {
         return locale.toString().replace('_', '-');
     }
+
+    public static List<String> getSensitiveHeaderList() {
```

Comment:
Please convert `SENSITIVE_HEADERS` to the contextual property:

```
private static final String SENSITIVE_HEADERS = "org.apache.http.sensitive.headers";


List<String> getSensitiveHeaders() {
    return MessageUtils.getContextualStrings(message, SENSITIVE_HEADERS, Set.of("Authorization", "Proxy-Authorization");
}
```

Reply:
Hello @reta 
Both `toString()` and `logProtocolHeaders` methods (who are the only users of `SENSITIVE_HEADERS` list) are static and don't use `message` to print headers.

Reply:
Hello @dmytro-sylaiev , that is right, those are internal APIs (package protected) and has to be changed to instance methods (or accept message as an argument), thanks.